### DISCUSSION TITLE
希望更改一下点击我的动态之后的反应

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#2215] [功能建议 / Ideas]
每次点击了我的动态里的一个视频后，动态就会自己合上消失，希望点击之后动态还在那里展示，等我点击空白处或者点击叉号后才自己合上。
![QQ截图20210825232922](https://user-images.githubusercontent.com/32454224/130819812-ff41d80d-4d88-4258-bab0-4567a82aa784.png)

**Comment by the1812** [comment]
按住 ctrl 再点

**Reply by ghost** [reply]
可以，但不够人性化

**Reply by Geeyun-JY3** [reply]
中键单击也是一样的效果

**Comment by Geeyun-JY3** [comment]
是希望前台新标签页打开视频，本标签页的这个动态面板回来后仍不收起吗？

**Reply by ghost** [reply]
对的，因为看自己关注的有好几个视频要看，点回来之后那个面板就自动关闭了，还要自己再点开。也可以加一个这个设置，这样不会影响到已有用户体验

**Reply by Geeyun-JY3** [reply]
感觉稍后再看就是针对你这需求的。
